packages/api/src/brevo-config/brevo-config.resolver.ts

```diff
@@ -93,8 +115,19 @@ export function createBrevoConfigResolver({
                 validateNotModified(brevoConfig, lastUpdatedAt);
             }
 
+            if (!input.senderMail || !input.senderName) {
+                throw new Error("Sender mail, sender name are required");
+            }
+
+            if (!(await this.isValidSender({ email: input.senderMail, name: input.senderName }))) {
+                throw new Error("Sender not found");
+            }
```

Comment:
The check for the valid template id is missing here

Reply:
Is this also included in the update, not only create?

Reply:
I added the validation for the template id here: https://github.com/vivid-planet/comet-brevo-module/pull/113/commits/88345b81fe1ceb38bd2d4d8ec0990b6627a2ea05
and also removed one of the validations for the sender in the update function, as it was there twice